src/server/serviceProvider/clients/community.client.js

```diff
@@ -1430,6 +1430,14 @@ function addGenres(endpoint, {reviewId, genres}) {
   });
 }
 
+function searchReviews(endpoint, {}) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this pattern to not be empty. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3799)